src/components/MapOverlay/MapOverlay.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Alert, Image } from 'react-native';
+import userDefaults from 'react-native-user-defaults';
+import LocationButton from '../LocationButton';
+import StationSlider from '../StationSlider';
+import {
+  AboutTouchableOpacity, ContainerView, HeaderContainerView,
+  HeaderView, HelpView, ModeSelectorSegmentedControlIOS,
+  OfflineButtonTouchableOpacity, OfflineText
+} from './MapOverlayCss';
+
+export default class MapOverlay extends React.Component {
+
+  static propTypes = {
+    connected: PropTypes.bool.isRequired,
+    error: PropTypes.string,
+    fetchNearest: PropTypes.func.isRequired,
+    loading: PropTypes.bool.isRequired,
+    locationDenied: PropTypes.bool.isRequired,
+    mode: PropTypes.string.isRequired,
+    nearestStationIndex: PropTypes.number, // eslint-disable-line
+    seeAllStations: PropTypes.func.isRequired,
+    showCallout: PropTypes.func.isRequired, // eslint-disable-line
+    stationDistances: PropTypes.array, // eslint-disable-line
+  }
+
+  state = {
+    displaySchedule: false,
```

Comment:
Not my code, I just kept what was already there. You could dive into why it's there and fix it though 👍.